lighthouse-core/report/report-generator.js

```diff
@@ -195,10 +195,10 @@ class ReportGenerator {
 
   /**
    * Gets the CSS for the report.
-   * @return {string}
+   * @return {Array<string>} an array of CSS
    */
   getReportCSS() {
-    return fs.readFileSync(path.join(__dirname, './styles/report.css'), 'utf8');
+    return [fs.readFileSync(path.join(__dirname, './styles/report.css'), 'utf8')];
```

Comment:
the isn't used to bring in more than one stylesheet, but was there a reason to make the change? I think it does make sense to move the perfX-specific styling to a separate file where possible